CMakeLists.txt

```diff
@@ -406,6 +430,13 @@ if (IS_64BIT_BUILD EQUAL 1)
   endif (CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64)
   add_definitions(-D_TARGET_AMD64_=1)
   add_definitions(-DDBG_TARGET_AMD64)
+elseif (CLR_CMAKE_PLATFORM_ARCH_ARM)
+  if (CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM)
+    add_definitions(-DDBG_TARGET_ARM_UNIX)
+  endif (CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM)
+  add_definitions(-D_TARGET_ARM_=1)
+  add_definitions(-DDBG_TARGET_ARM)
+  add_definitions(-DSTUBLINKER_GENERATES_UNWIND_INFO)
```

Comment:
Do we need to keep the definition order in StubLinkCPU or am I okay to change the order of definitions in that class?